colcon_bundle/verb/utilities.py

```diff
@@ -9,10 +9,39 @@
 import subprocess
 import sys
 
-
 from colcon_bundle.verb import logger
 
 
+def get_ros_distribution_version():
+    """
+    Discover and return ROS distribution version.
```

Comment:
Would be great to explain why the resolution is done this way. This seems counter-intuitive.
I could have compiled ROS X Y Z on Ubuntu Bionic, no?